packages/sdk-core/src/Operation.ts

```diff
@@ -18,12 +18,18 @@ export default class Operation {
     readonly populateTransactionPromise: Promise<ethers.PopulatedTransaction>;
     readonly type: OperationType;
 
+    // @note This property is used to ensure BatchCall operations still function
+    // when using the agreement forwarder
+    readonly forwarderPopulatedPromise?: Promise<ethers.PopulatedTransaction>;
```

Comment:
Ideally a factory function (`() => ...`) would be better for `forwarderPopulatedPromise` & `populateTransactionPromise` to avoid always evaluating both of the promises.

Reply:
agreed, we can iterate on this in the future, just wanted to push some of this stuff out ASAP so @tokdaniel could demo his stuff today